website/source/guides/identity/identity.html.md

```diff
@@ -0,0 +1,1123 @@
+---
+layout: "guides"
+page_title: "Identity: Entities and Groups - Guides"
+sidebar_current: "guides-identity-identity"
+description: |-
+  This guide demonstrates the commands to create entities, entity aliases, and
+  groups.  For the purpose of the demonstration, userpass auth method will be
+  used.  
+---
+
+# Identity - Entities and Groups
+
+Vault supports multiple authentication methods and also allows enabling the same
+type of authentication method on different mount paths. Each Vault client may
+have multiple accounts with various identity providers that are enabled on the
+Vault server.
+
+Vault clients can be mapped as ***entities*** and their corresponding accounts
+with authentication providers can be mapped as ***aliases***. In essence, each
+entity is made up of zero or more aliases. Identity secrets engine internally
+maintains the clients who are recognized by Vault.
+
+## Reference Material
+
+- [Identity Secrets Engine](/docs/secrets/identity/index.html)
+- [Identity Secrets Engine (API)](/api/secret/identity/index.html)
+- [External vs Internal Groups](/docs/secrets/identity/index.html#external-vs-internal-groups)
+
+~> **NOTE:** An [interactive
+tutorial](https://www.katacoda.com/hashicorp/scenarios/vault-identity) is
+also available if you do not have a Vault environment to perform the steps
+described in this guide.
+
+## Estimated Time to Complete
+
+10 minutes
+
+## Personas
+
+The steps described in this guide are typically performed by **operations**
```

Comment:
Are these personas defined centrally? If so, can we link to them?

Reply:
At the moment, there is no definition other than what people normally think **ops** people to be in organizations.  When we roll out the online education program, this guide will appear under **Operations**' learning _track_.